.github/workflows/docker-image2.yml

```diff
@@ -0,0 +1,44 @@
+name: ZZZZ
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+
+
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to Custom registry 
+        uses: docker/login-action@v1 
+        with:
+          registry: cloud.canister.io:5000
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.REGISTRY_PASSWORD}}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: |
+            cloud.canister.io:5000/locohamster/loco_exporter
+      - name: Print image url
+        run: echo "Image pushed to ${{ steps.push-to-quay.outputs.registry-paths }}"
```

Comment:
this should stay in your fork, not in a PR.

Reply:
oh sorry, I was currently working on this. this should have been branch used for PR...